multisig/simulations/arbitrumsepolia/vip-015/index.ts

```diff
@@ -0,0 +1,371 @@
+            expect(await vToken.reserveFactorMantissa()).to.equal(parseUnits(params.reserveFactor, 18));
+          });
+
+          it(`should set ${symbol} collateral factor to ${params.collateralFactor}`, async () => {
+            const market = await comptroller.markets(vTokens[symbol]);
+            expect(market.collateralFactorMantissa).to.equal(parseUnits(params.collateralFactor, 18));
+          });
+
+          it(`should set ${symbol} liquidation threshold to ${params.liquidationThreshold}`, async () => {
+            const market = await comptroller.markets(vTokens[symbol]);
+            expect(market.liquidationThresholdMantissa).to.equal(parseUnits(params.liquidationThreshold, 18));
+          });
+
+          it(`should set ${symbol} protocol seize share to 0.01`, async () => {
+            expect(await vToken.protocolSeizeShareMantissa()).to.equal(parseUnits("0.01", 18));
+          });
+
+          it(`should set ${symbol} supply cap to ${params.supplyCap}`, async () => {
+            expect(await comptroller.supplyCaps(vTokens[symbol])).to.equal(
+              parseUnits(params.supplyCap, underlyingDecimals),
+            );
+          });
+
+          it(`should set ${symbol} borrow cap to ${params.borrowCap}`, async () => {
+            expect(await comptroller.borrowCaps(vTokens[symbol])).to.equal(
+              parseUnits(params.borrowCap, underlyingDecimals),
+            );
+          });
+        });
+      }
+    });
+
+    describe("Pools configuration", () => {
+      const checkComptroller = (comptrollerAddress: string, comptrollerName: string) => {
+        describe(`${comptrollerName} Comptroller`, () => {
+          let comptroller: Contract;
+
+          before(async () => {
+            comptroller = await ethers.getContractAt(COMPTROLLER_ABI, comptrollerAddress);
+          });
+
+          it("should use the correct comptroller address", async () => {
+            expect(comptroller.address).to.equal(comptrollerAddress);
+          });
+
+          it("should have the correct price oracle", async () => {
+            expect(await comptroller.oracle()).to.equal(RESILIENT_ORACLE);
+          });
+
+          it("should have close factor = 0.5", async () => {
+            expect(await comptroller.closeFactorMantissa()).to.equal(parseUnits("0.5", 18));
+          });
+
+          it("should have liquidation incentive = 1.1", async () => {
+            expect(await comptroller.liquidationIncentiveMantissa()).to.equal(parseUnits("1.02", 18));
+          });
+
+          it("should have minLiquidatableCollateral = $100", async () => {
+            expect(await comptroller.minLiquidatableCollateral()).to.equal(parseUnits("100", 18));
+          });
+
+          it("should have owner = GUARDIAN", async () => {
+            expect(await comptroller.owner()).to.equal(GUARDIAN);
+          });
+        });
+      };
+
+      checkComptroller(COMPTROLLER_LIQUID_STAKED_ETH, "Liquid Staked ETH");
+    });
+
```

Comment:
Could we use `checkIsolatedPoolsComptrollers`?

Reply:
`checkIsolatedPoolsComptrollers` is configured only for BNB chain